### PR TITLE
Version upgrade f651d45

### PR DESCRIPTION
**Pull request by haerdib**

### CODE REVIEW

File: substrate-sgx/sp-io/Cargo.toml [hunk 1]
```diff
@@ -18,58 +18,19 @@ environmental = { version = "1.0.1", default-features = false, optional = true }
...
+sp-core = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", features=["full_crypto"], branch = "master" }
+sp-state-machine = { version = "0.10.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "master", optional = true}
+sp-runtime-interface = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master" }
+sp-wasm-interface = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master" }
+sp-tracing = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master" }
+sp-trie = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "master", optional = true }
+sp-keystore = {  version = "0.10.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master", optional = true }
+sp-externalities = { version = "0.10.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "master", optional = true }
```
Comment (clangenb) [inline substrate-sgx/sp-io/Cargo.toml:32]: I love you

**Review by clangenb — APPROVED** [review]
Looks good to me, but I suggest we test it first with the worker.

File: substrate-sgx/sp-io/Cargo.toml (hunk 1 above)
Comment (haerdib) [reply substrate-sgx/sp-io/Cargo.toml:32]: Agreed. I will test it locally with the worker first. I'll notify you as soon as that is done.

**Comment by haerdib** [comment]
https://github.com/integritee-network/worker/pull/347

Successfully passed. Can we merge?

**Review by brenzi — APPROVED** [review]

**Comment by clangenb** [comment]
Yes, merge.

**Comment by haerdib** [comment]
@brenzi, can you merge? @clangenb  and I are not authorized

**Pull request merged**